packages/stark-ui/src/modules/toast-notification/services/toast-notification.service.ts

```diff
@@ -23,7 +23,7 @@ export class StarkToastNotificationServiceImpl implements StarkToastNotification
 	/**
 	 * Observer linked to the currently displayed toast notification
 	 */
-	public currentToastResult$?: Observer<StarkToastNotificationResult>;
+	private currentToastResult$?: Observer<StarkToastNotificationResult>;
```

Comment:
Why changing this from `public` to `private` ?

Reply:
because we refactor and this due to the fact the Oberver interface do not contain closed property anymore